README.md

```diff
@@ -59,29 +59,29 @@ device respectively. In order to run this demo, the following hardware is requir
 4. Select desired image:
 
    - click **CHOOSE OS** button:
-      ![Step 1](doc/pictures/install_step_1.png)
```

Comment:
Regardless md supports html tags, the native way to add new line it is just add empty line before and after pic link.